static_src/test/unit/components/route.spec.jsx

```diff
@@ -0,0 +1,60 @@
+
+import '../../global_setup.js';
+
+import React from 'react';
+import { shallow } from 'enzyme';
```

Comment:
I guess this lets you render react components so they don't render all subcomponents? cool